plugins/db-mongo/src/index.ts

```diff
@@ -101,7 +99,20 @@ class MongoPlugin implements AmplicationPlugin {
     context: DsgContext,
     eventParams: CreateServerDotEnvParams
   ) {
-    eventParams.envVariables = [...eventParams.envVariables, ...envVariables];
+    const { settings } = currentInstallation(context.pluginInstallations);
+    const { port, password, user, host, dbName } = settings;
+
+    eventParams.envVariables = [
+      ...eventParams.envVariables,
+      ...[
+        { DB_USER: user },
+        { DB_PASSWORD: password },
+        { DB_PORT: port },
+        {
+          DB_URL: `mongodb://${user}:${password}@${host}:${port}/${dbName}?authSource=admin`,
+        },
+      ],
```

Comment:
why did you change it? it is exactly as it was.

Reply:
Have a look at the values, it's not a string, its the values themselves.

Reply:
but you can still use the constants as we did in other palaces to be consistent

Reply:
No we cant, because this values come from the plugin settings